### PR TITLE
Clarify

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/Viva/issues/501

### CODE REVIEW

**Comment by get-itips** [comment]
@JohanFreelancer9

File: Viva/learning/configure-sharepoint-content-source.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ To configure SharePoint as a learning content sources in for Viva Learning, foll
 
      ![SharePoint list showing the Site contents navigation and the Learning App Content Repository section.](../media/learning/learning-sharepoint-configure4.png)
 
-5. On the **Learning App Content Repository** page, populate the SharePoint list with URLs to the learning content folders.
+5. On the **Learning App Content Repository** page, populate the SharePoint list with URLs to the learning content folders. Read [Folder URL document library curation](#folder-url-document-library-curation) on how to create the content folders.
```
Comment (JohanFreelancer9) [inline Viva/learning/configure-sharepoint-content-source.md:86]:
```suggestion
5. On the **Learning App Content Repository** page, populate the SharePoint list with URLs to the learning content folders. Read [Folder URL document library curation](#folder-url-document-library-curation) for details about how to create the content folders.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@bhaswatic 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**